homework_2/task4.c

```diff
@@ -0,0 +1,48 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+#define len 15
+
+void sort(int ar[], int ln)
```

Comment:
```suggestion
void partition(int array[], int lenght)
```
Эта функция не сортирует, а разделяет массив.
 И сокращения долой.